protocol/v2/ssv/runner/proposer.go

```diff
@@ -139,6 +164,7 @@ func (r *ProposerRunner) ProcessPreConsensus(ctx context.Context, logger *zap.Lo
 	logger.Info("🧊 got beacon block proposal",
 		zap.String("block_hash", blockSummary.Hash.String()),
 		zap.Bool("blinded", blockSummary.Blinded),
+		zap.Duration("mev_delay", r.mevDelay),
 		zap.Duration("took", time.Since(start)),
```

Comment:
To improve clarity in logs related to block fetching, consider replacing:
```go
zap.Duration("took", time.Since(start)),
```
with:
```go
zap.Duration("block_fetch_time", time.Since(start) - r.mevDelay),
zap.Duration("total_time", time.Since(start)),
```